src/app/beer_garden/api/http/__init__.py

```diff
@@ -76,6 +76,117 @@
 client_ssl: ssl.SSLContext
 
 
+def _get_published_url_specs(
+    prefix: Optional[str] = None,
+) -> List[Tuple[str, RequestHandler]]:
+    """Generates the urls specs for the tornado Application that are intended to be
+    published in the swagger docs
+    """
+    if prefix is None:
+        prefix = config.get("entry.http.url_prefix")
+
+    return [
+        # V1
+        (rf"{prefix}api/v1/requests/?", v1.request.RequestListAPI),
+        (rf"{prefix}api/v1/systems/?", v1.system.SystemListAPI),
+        (rf"{prefix}api/v1/queues/?", v1.queue.QueueListAPI),
+        (rf"{prefix}api/v1/users/?", v1.user.UserListAPI),
+        (rf"{prefix}api/v1/admin/?", v1.admin.AdminAPI),
+        (rf"{prefix}api/v1/jobs/?", v1.job.JobListAPI),
+        (rf"{prefix}api/v1/gardens/?", v1.garden.GardenListAPI),
+        (rf"{prefix}api/v1/namespaces/?", v1.namespace.NamespaceListAPI),
+        (rf"{prefix}api/v1/instances/(\w+)/?", v1.instance.InstanceAPI),
+        (rf"{prefix}api/v1/instances/(\w+)/logs/?", v1.instance.InstanceLogAPI),
+        (rf"{prefix}api/v1/instances/(\w+)/queues/?", v1.instance.InstanceQueuesAPI),
+        (rf"{prefix}api/v1/requests/(\w+)/?", v1.request.RequestAPI),
+        (rf"{prefix}api/v1/requests/output/(\w+)/?", v1.request.RequestOutputAPI),
+        (rf"{prefix}api/v1/systems/(\w+)/commands/(\w+)/?", v1.command.CommandAPI),
+        (rf"{prefix}api/v1/systems/(\w+)/?", v1.system.SystemAPI),
+        (rf"{prefix}api/v1/queues/(.*)/?", v1.queue.QueueAPI),
```

Comment:
Other than the refactor for test purposes, this is the only line that actually needed to change for this PR.